lte/gateway/docker/services/build/freediameter.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/third_party/build/bin/freediameter_build.sh b/third_party/build/bin/freediameter_build.sh
```

Comment:
Can you discuss the source of these patches / need in the PR description maybe? Nothing long but so we have context. There are existing patches for some dependencies in Tim's [third-party/build/patches](https://github.com/magma/magma/tree/master/third_party/build/patches) and I would like to know these are not duplicated here.. or if so why they have to be duplicated.

Reply:
Following up on this comment - redundant files will be dropped from the PR after discussion with @rmeleromira

Reply:
removed